libcomposefs/lcfs-writer.c

```diff
@@ -564,10 +564,9 @@ int lcfs_compute_fsverity_from_fd(uint8_t *digest, int fd)
 	return lcfs_compute_fsverity_from_content(digest, &_fd, fsverity_read_cb);
 }
 
-// Given a file descriptor, first query the kernel for its fsverity digest.  If
-// it is not available in the kernel, perform an in-memory computation.  The file
-// position will always be reset to zero if needed.
-int lcfs_fd_get_fsverity(uint8_t *digest, int fd)
+// Given a file descriptor, query the kernel for its fsverity digest. It
+// is an error if fsverity is not enabled.
+int lcfs_fd_require_fsverity(uint8_t *digest, int fd)
```

Comment:
I think the better verb here is `measure` to match the name of the ioctl.

Then we'd have:
 - `get`: get it for me, don't care how
 - `compute`: actually do the work in full, in userspace
 - `measure`: use the measure ioctl

Reply:
Good suggestion, fixed!